models/User.go

```diff
@@ -1386,20 +1393,22 @@ func (e WayTooMuchCashError) Error() string {
 	return "You already have more than enough cash. You cannot mortgage stocks right now."
 }
 
-func PerformMortgageTransaction(userId, stockId uint32, stockQuantity int64) (*Transaction, error) {
+// PerformMortgageTransaction returns mortgage transaction and/or error
+func PerformMortgageTransaction(userId, stockId uint32, stockQuantity int64, retrievePrice uint64) (*Transaction, error) {
 	var l = logger.WithFields(logrus.Fields{
 		"method":              "PerformMortgageTransaction",
 		"param_userId":        userId,
 		"param_stockId":       stockId,
 		"param_stockQuantity": stockQuantity,
+		"param_retrievePrice": retrievePrice,
 	})
 
-	l.Infof("PerformMortgageTransaction requested")
+	l.Infof("PerformMortgageTransaction requested for userId = %d, stockId = %d amount = %d retrievePrice = %d", userId, stockId, stockQuantity, retrievePrice)
```

Comment:
Remove the parameters, they'll be printed automatically because `l` is created `WithFields` to print those fields for every statement l prints.